eval_test.go

```diff
@@ -167,7 +167,8 @@ func load(thread *skylark.Thread, module string) (skylark.StringDict, error) {
 	}
 
 	// TODO(adonovan): test load() using this execution path.
-	filename := filepath.Join(filepath.Dir(thread.Caller().Position().Filename()), module)
+	filename := filepath.Join(filepath.Dir(thread.
```

Comment:
Seems like an odd place to break the line. Was it too long before? Extracting the directory into a variable might be slightly better.

Reply:
This newline was introduced by mistake; removed.